### DISCUSSION TITLE
Question about motorPowerSet parameter

### DISCUSSION THREAD

**Discussion by Xinlyu123** [bitcraze/discussions#156] [Q&A]
For 4 motors I used different value to make the motor to start spin. M1 with value 19000 M2,M3,M4 with value 7000. Is this normal? Because I saw the post about the parameter around 3000 is enough to make the motor to spin.
Is 4 motors should have same parameter value to start spinning? When I use gamepad, the drone flipped, so I am wondering 4 motors require different parameter can be the potential reason that cause this issue. (I already check the propeller and motor spin direction).

**Comment by Xinlyu123** [comment]
Additionally, I found the m1 motor is spinning earlier or spinning faster than the other motors.

**Comment by knmcguire** [comment]
Hi!

It could be that the m1 connector is not properly connecting in that case? Perhaps due to some residue, glue or something else. You can try to rub that off by connection and reconnecting that motor's connector several times, and if that doesn't work try to put a tiny drop of wb40 on the connector pins for better flow. Also make sure that that connector is not secretly lose or something if you look between the control board and the white male connector itself.

**Comment by Xinlyu123** [comment]
I think I got it four motors have different parameter value is fine. I just need tuning the pid value to make the flight stable. Thanks!
